compiler/passes/checkParsed.cpp

```diff
@@ -463,6 +463,16 @@ checkFunction(FnSymbol* fn) {
       numNonVoidReturns == 0) {
     USR_FATAL_CONT(fn, "function declared 'ref' but does not return anything");
   }
+
+  if (isIterator) {
+    for_formals(formal, fn) {
+      if (formal->intent == INTENT_OUT) {
+        USR_FATAL_CONT(formal, "out intent not yet supported for iterators");
```

Comment:
Going forward, I would like us to use English in cases like this, i.e. using "is" or another verb .
OK to leave as-is for this PR

Reply:
I'll add *is*, no worries